resize.js

```diff
@@ -0,0 +1,76 @@
+var preserve_ratio = "%(preserve_ratio)s"; // string
+var min_height = %(min_height)d;
```

Comment:
Because this is not a valid javascript (before injection), these statements throw out an error in a code editor. Perhaps we could wrap it in a string and convert it to an int?

Reply:
I must admit that I didn't have this problem in emacs, but I see no problem with that.

FYI, in the add-on "resize image in editor", I have choosen to put all variables to null. Then to evaluate code which change the variables's value in the python method. So I never really had any similar problem.
This is useful mostly because it allows to change the configuration without having to restart anki. Those new values will be updated as soon as the editor (or reviewer in this case) is closed and reopened. But it seems you're not interested in this feature

Reply:
I think that's actually a good idea! The current code is mostly an artifact from having js as a string in the __init__.py, then moving it to its own file. Definitely something to consider later

In this case, the config is indeed updated each time reviewer is closed/opened, because config is retrieved in `myRevHtml`, which is called whenever reviewer is opened, or each 100 reviewes when web is reinitialized.